Query.php

```diff
@@ -165,6 +165,12 @@ class Query extends Component implements QueryInterface
      * @since 2.0.4
      */
     public $options = [];
+    /**
+     * @var string|array The post filter query for differentially filter search results and aggregations
+     * @see https://www.elastic.co/guide/en/elasticsearch/guide/current/_post_filter.html
+     * @since 2.0.5
+     */
+    public $postFilter = [];
```

Comment:
should  be default null, meaning it is not added if not specified. See `$filter`. Maybe even move it below `$filter`.